conversions/octal_to_decimal

```diff
@@ -0,0 +1,37 @@
+def oct_to_decimal(oct_string: str) -> int:
+    """
+    Convert a octal value to its decimal equivalent
+
+    >>> oct_to_decimal("12")
+    10
+    >>> oct_to_decimal(" 12   ")
+    10
+    >>> oct_to_decimal("-45")
+    -37
+    >>> oct_to_decimal("2-0Fm")
+    ValueError: Non-octal value was passed to the function
+    >>> oct_to_decimal("")
+    ValueError: Empty string value was passed to the function
+    >>> oct_to_decimal("19")
+    ValueError: Non-octal value was passed to the function
+    """
+    oct_string = oct_string.strip()
```

Comment:
```suggestion
    oct_string = str(oct_string).strip()
```
Do the right thing if the user passes an int instead of a string...  oct_to_decimal(-45) vs. oct_to_decimal("-45")